components/list/list.css

```diff
@@ -0,0 +1,61 @@
+/*** List Spaced ***/
+
+.list{
+    background-color: var(--Elevation-00);
+    list-style: none;
+    padding-left: 0;
+    margin:0;
+    padding: .8rem 1rem ;
+    display: flex;
+    flex-direction: column;
+    gap:.25rem;
+    border-radius: .25rem;
+    width: 100%;
+    box-shadow: var(--Shadow-01);
+}
+.list-item{
+    box-shadow: var(--Shadow-01);
+    background-color: var(--Elevation-01);
+    display: flex;
+    align-items:center ;
+    gap:10%;
```

Comment:
On lines 11 and 21, try using the same unit for the property `gap`.  

Rest all looks good!

Reply:
ok noted 👍